samples/hosting/docker/sample.md

```diff
@@ -12,6 +12,15 @@ redirects:
 
 This sample demonstrates how to use Docker Linux containers to host NServiceBus endpoints communicating over the [RabbitMQ transport](/transports/rabbitmq/). While this sample uses [Docker Compose](https://docs.docker.com/compose/) to demonstrate how to orchestrate a multi-container application, the containers are compatible withe other orchestration technologies, for example [Kubernetes](https://kubernetes.io/docs/home/).
 
+The endpoints use the [.NET SDK Container Building Tools](https://github.com/dotnet/sdk-container-builds) to enable the creation of containers via the `dotnet publish` command. See the [Microsoft tutorial](https://learn.microsoft.com/en-us/dotnet/core/docker/publish-as-container?pivots=dotnet-8-0) and [customization documentation](https://github.com/dotnet/sdk-container-builds/blob/main/docs/ContainerCustomization.md) for more details.
+
+## Prerequisites
+
+To containerize a .NET app using `dotnet publish`:
+
+- **[.NET 8+ SDK](https://dotnet.microsoft.com/download/dotnet/8.0)**: Check your installed SDK version with `dotnet --info`.
```

Comment:
The original 8.0 SDK didn't do this without the NuGet package. [As noted here](https://learn.microsoft.com/en-us/dotnet/core/docker/publish-as-container?pivots=dotnet-8-0#add-nuget-package). We should be more specific.
```suggestion
- **[.NET 8.0.200 SDK](https://dotnet.microsoft.com/download/dotnet/8.0)** (or higher): Check your installed SDK version with `dotnet --info`.
```